### PR TITLE
Group container run output for GHA

### PR DESCRIPTION
**Pull request by mkolesnik**
Usually we don't really care to see the whole info of the Shipyard container invocation on the log in GHA.

This will group it in a collapsible group for easier log reading on the CI.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: scripts/shared/entry [hunk 1]
```diff
@@ -15,4 +15,5 @@ else
 fi
 
 shift
+[[ -z "$GITHUB_ACTOR" ]] || echo "::endgroup::"
```
Comment (skitt) [inline scripts/shared/entry]:
```suggestion
# This ends the group started in Makefile.dapper
[[ -z "$GITHUB_ACTOR" ]] || echo "::endgroup::"
```

**Review by skitt — CHANGES_REQUESTED** [review]
Let’s add a comment to point to where the group starts.

**Review by skitt — APPROVED** [review]

**Comment by mkolesnik** [comment]
I changed the wording to make more sense when looking in the GHA log 
image

**Comment by mkolesnik** [comment]
Seems that relying on various `GITHUB_` variables to be set is problematic, added the `CI` to be passed down so that we can definitively identify a CI run.

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**